ffi/derive/src/attr_parse/repr.rs

```diff
@@ -136,7 +136,7 @@ impl FromAttributes for Repr {
                     ),
                     Meta::List(list) => {
                         let Some(tokens) = accumulator.handle(
-                            syn2::parse2::<ReprTokens>(list.tokens.clone()).map_err(Into::into)
+                            syn2::parse2::<ReprTokens>(list.tokens.clone()).map_err(Into::into),
```

Comment:
Looks like you used `cargo fmt` of a newer version or something, which changed a lot of file. Just wondering, no objections